packages/dev/core/src/Engines/ICanvas.ts

```diff
@@ -112,6 +112,48 @@ export interface ITextMetrics {
     readonly actualBoundingBoxRight: number;
 }
 
+/**
+ * Class used to abstract a matrix
+ */
+export interface DOMMatrix {
+    /**
+     * A Boolean flag whose value is true if the matrix was initialized as a 2D matrix. If false, the matrix is 3D.
+     */
+    is2D: boolean;
+    /**
+     * A Boolean whose value is true if the matrix is the identity matrix. The identity matrix is one in which every value is 0 except those on the main diagonal from top-left to bottom-right corner (in other words, where the offsets in each direction are equal).
+     */
+    isIdentity: boolean;
+    /**
+     * The following double-precision floating-point values represent the components of a 4×4 matrix which are required in order to perform 2D rotations and translations.
+     */

```

Comment:
I think you meant a 3x2 matrix.

Reply:
From my understanding, the DOMMatrix class is 4x4: https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix, through I can eliminate the doubt by just saying matrix, I think?